website/pages/en/billing.mdx

```diff
@@ -2,10 +2,32 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency through Banxa](#billing-on-banxa)
+- [Paying with GRT on Arbitrum](#billing-on-arbitrum)
+
+## Billing on Banxa
+
+Banxa enables you to bypass the need for an exchange and pay for your query fees using the fiat currency of your choice. The fiat currency will be converted to GRT, added to your billing balance on the billing contract, and used to pay for your query fees.
```

Comment:
We repeat "pay for your query fees" twice in this paragraph. I wonder if one could be removed, or use "pay for your API keys" instead